app/router.go

```diff
@@ -155,4 +179,16 @@ type topologyView struct {
 	human    string
 	parent   string
 	renderer render.Renderer
+	options  optionParams
+}
+
+type optionParams map[string][]optionValue // param: values
+
+type optionValue struct {
```

Comment:
Is there a way to not have APITopologyOption and this struct?  Some cleve combinations of struct tags might be able to give the same result.